src/Symfony/Component/Form/Tests/Extension/Core/Type/ChoiceTypeTest.php

```diff
@@ -94,6 +100,54 @@ public function testExpandedChoicesOptionsTurnIntoChildren()
         $this->assertCount(count($this->choices), $form, 'Each choice should become a new field');
     }
 
+    public function testChoiceListWithScalarValues()
+    {
+        $view = $this->factory->create('Symfony\Component\Form\Extension\Core\Type\ChoiceType', null, array(
+            'choices' => $this->scalarChoices,
+        ))->createView();
```

Comment:
@iltar, I know, I even proposed to use constants in core type tests (see https://github.com/symfony/symfony/pull/17699#discussion_r51989703) as of 2.8 but this is actually to be consistent with other tests ;)